crates/l3/appchain-core-contract-client/src/interfaces/operator.rs

```diff
@@ -1,6 +1,8 @@
-use color_eyre::{eyre::eyre, Result};
+// allow unused imports
+#![allow(unused_imports)]
```

Comment:
Why ?

Reply:
it was done earlier as Provider was not being identified by client, but it's now working, removed!